docs/operations/metrics.md

```diff
@@ -326,6 +326,12 @@ If `emitBalancingStats` is set to `true` in the Coordinator [dynamic configurati
 
 ## General Health
 
+### Overlord/Coordinator
```

Comment:
I think it can be used for all services, even though it might not be very useful right now except for coordinator and overlord.

Reply:
Updated.